{"name": "test_del_user_from_group", "status": "broken", "statusDetails": {"message": "pony.orm.core.UnrepeatableReadError: Phantom object ORMUsers[5] disappeared", "trace": "app = <fixture.application.Application object at 0x108d003d0>, db = <fixture.db.DbFixture object at 0x108d00550>\norm = <fixture.orm.ORMFixture object at 0x108d45040>\n\n    def test_del_user_from_group(app, db, orm):\n        group = db.get_group_list()\n        for x in group:\n>           if orm.get_users_in_group(x) == []:\n\ntest/test_del_user_from_group.py:10: \n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ \n<string>:2: in get_users_in_group\n    ???\nvenv/lib/python3.8/site-packages/pony/orm/core.py:552: in new_func\n    reraise(exc_type, exc, tb)\nvenv/lib/python3.8/site-packages/pony/utils/utils.py:95: in reraise\n    try: raise exc.with_traceback(tb)\nvenv/lib/python3.8/site-packages/pony/orm/core.py:533: in new_func\n    result = func(*args, **kwargs)\nfixture/orm.py:53: in get_users_in_group\n    return self.convert_users_to_model(orm_group.users)\nfixture/orm.py:48: in convert_users_to_model\n    return list(map(convert, users))\nfixture/orm.py:47: in convert\n    return Users(id=str(user.id), users_name=user.firstname, users_lastname=user.lastname)\nvenv/lib/python3.8/site-packages/pony/orm/core.py:2288: in __get__\n    value = attr.get(obj)\nvenv/lib/python3.8/site-packages/pony/orm/core.py:2298: in get\n    val = vals[attr] if attr in vals else attr.load(obj)\nvenv/lib/python3.8/site-packages/pony/orm/core.py:2282: in load\n    else: obj._load_()\nvenv/lib/python3.8/site-packages/pony/orm/core.py:4833: in _load_\n    if obj not in objects: throw(UnrepeatableReadError,\n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ \n\nexc_type = <class 'pony.orm.core.UnrepeatableReadError'>, args = ('Phantom object ORMUsers[5] disappeared',), kwargs = {}\n\n    def throw(exc_type, *args, **kwargs):\n        if isinstance(exc_type, Exception):\n            assert not args and not kwargs\n            exc = exc_type\n        else: exc = exc_type(*args, **kwargs)\n        exc.__cause__ = None\n        try:\n            if not (pony.MODE == 'INTERACTIVE' and options.CUT_TRACEBACK):\n>               raise exc\nE               pony.orm.core.UnrepeatableReadError: Phantom object ORMUsers[5] disappeared\n\nvenv/lib/python3.8/site-packages/pony/utils/utils.py:106: UnrepeatableReadError"}, "start": 1613509063514, "stop": 1613509063519, "uuid": "1768d43c-b125-4ee0-94b0-e0fb118a3139", "historyId": "c239c10dbf24dd332473714a089617c8", "testCaseId": "a9726a0a65a3ecf1926d4db68d91e7bb", "fullName": "test.test_del_user_from_group#test_del_user_from_group", "labels": [{"name": "parentSuite", "value": "test"}, {"name": "suite", "value": "test_del_user_from_group"}, {"name": "host", "value": "MacBook-Pro-Anna.local"}, {"name": "thread", "value": "97099-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "test.test_del_user_from_group"}]}